pages/posts/2.tsx

```diff
@@ -126,16 +122,16 @@ func main() {
 go run .`}</code>
                 </pre>
                 <p>
-                  (You only need to do <code className="language-js token string">go mod tidy</code> the first time.) It should print something like{" "}
+                  (You only need to do <code>go mod tidy</code> the first time.) It should print something like{" "}
```

Comment:
Can you just put the response “Hello, world! My peer ID is 12D3KooWGVMm5mum6qMtwbReiD9Aa93WTuTT2hTHsYxQASTzYUfs” in quotes as normal text, without a code block? For me the new code style gives it weird colors.